### PR TITLE
Refactor Anthropic import to langchain_anthropic and update model to v3

### PR DESCRIPTION
**Pull request by donbr**
- Transition Anthropic API import to the langchain_anthropic package for enhanced compatibility.
- Upgrade the AI model to claude-3-sonnet-20240229 for improved performance and features.

### CODE REVIEW

**Comment by donbr** [comment]
The contributor license agreement has been signed.

File: pyproject.toml [hunk 1]
```diff
@@ -18,6 +18,7 @@ httpx-sse = {version = ">=0.3.1", optional = true}
 pydantic = ">=1"
 langchain = ">=0.0.333"
 orjson = ">=2"
+langchain-anthropic = "^0.1.4"
```
Comment (eyurtsev) [inline pyproject.toml:21]: This change means that all developers that install LangServe will be forced to install anthropic as well. Would you be able to undo the changes in the poetry and pyproject files?

File: examples/chat_playground/server.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
-from langchain.chat_models import ChatAnthropic
+from langchain_anthropic.chat_models import ChatAnthropic
```
Comment (eyurtsev) [inline examples/chat_playground/server.py:9]: thank you!

Comment (donbr) [reply examples/chat_playground/server.py:9]: changes to the poetry and pyproject files have been rolled back.

File: examples/chat_playground/server.py [hunk 3]
```diff
@@ -5,7 +5,8 @@
 from typing import List, Union
 
 from fastapi import FastAPI
-from langchain.chat_models import ChatAnthropic
+from fastapi.middleware.cors import CORSMiddleware
```
Comment (eyurtsev) [inline examples/chat_playground/server.py:8]:
```suggestion
```

**Review by eyurtsev — APPROVED** [review]
